did-git-spec.md

```diff
@@ -125,29 +125,36 @@ Example:
   }
 }
 ```
+The commit adding your DID document should be signed by the key associated private key of the public key referenced in the DID document.
 
 ## Read
 
 Notes: since we don't know the repo did string when the repo DID document is created, the repo DID document will not contain the "id" did string for the repo. The `git did read` operation will "resolve" the "id" by looking up the SHA1 hash of the commit that added the DID document to the repo and dynamically add the "id" member to what is rendered to the user.
 
 ## Update 
-- repo.did
-    - updating service_endpoints?
-- [personal-hash].did
-    - is any signature/crypto verification required to update a [personal-hash].did, or, for example, can I change your keys and commit it to the repo to effectively lock you out?  Or is change to .did/[personal-hash].did only allowed where [personal-hash].did matches the signer?
+**1. Repository DID**
+
+There aren't currently any keys associated with the respository, so there will ne no need to update the Public Keys in the DID document.  The process to change the maintainers or the canonical endpoint should be defined in the `.did/governance.md` file.
+
+**2. Contributor DID**
+
+To update a Contributor DID document, remove the previous DID document from the repository and commit.  This commit should be signed by the private key of the DID document being removed.  Then create a new DID document with the appropriate information and commit to the repository.  This commit should be signed by the private key of the DID document being added.
+
+TODO:
+
+1. We need a way to verify that the person making each of these changes has the appropriate keys to prevent unauthorized updates.  Is there a cleaner way to do this?
 
 ## Delete (Revoke)
 
-In the context of a repository, deletion takes two forms.
-    - Deletion of the repository did itself
-    - Deletion of a specific did relative to the reposistory
+**1. Repository DID**
+
+The DID for the repository can be deleted by removing the `.did/repo.did` file from the respository and performaing a commit.  The signature of this commit should correspond to one of the maintainers in the `.did/repo.did#authrntication list. 
```

Comment:
do you mean `.did/repo.did#authorization` list?

Reply:
Should be authentication list.  I believe this is where we agreed to store the list of maintainers.